### PR TITLE
feat: update acknowledgments

### PR DESCRIPTION
**Pull request by MalinAhlberg**
This PR addresses [issue #16](https://github.com/NBISweden/SysDev-Maple/issues/16).

The acknowledgment is updated to 

 > This work used the <a href="https://www.egi.eu/infrastructure/cloud/">EGI infrastructure</a> with the support of IN2P3-IRES, INFN-CLOUD-BARI and TR-FC1-ULAKBIM.

### CODE REVIEW

File: proj/pred/templates/pred/submit_seq.html [hunk 1]
```diff
@@ -108,8 +108,7 @@ <h2>PRODRES: Protein Domain Reduced Search</h2>
      <tr><td> <h3>Reference</h3> </td></tr>
          {% include "pred/refbib.html" %}
      <tr><td> <h3>Resource provider</h3> </td></tr> 
-     <tr><td>PRODRES is supported by the <a href="https://www.egi.eu/services/cloud-compute/">EGI FedCloud</a> with the VO VO.NBIS.SE for computational resource. </td></tr> 
-     </td></tr>
+       {% include "pred/egi_acknowledgement.html" %}
```
Comment (nanjiangshu) [inline proj/pred/templates/pred/submit_seq.html:111]:
```suggestion
     <tr><td> 
       <h3>Resource provider</h3>
       {% include "pred/egi_acknowledgement.html" %}
     </td></tr>
```

**Review by nanjiangshu — COMMENTED** [review]
Just a little comment

File: proj/pred/templates/pred/submit_seq.html (hunk 1 above)
Comment (MalinAhlberg) [reply proj/pred/templates/pred/submit_seq.html]: Good that you noticed this! Fixed in 34c3f77

Comment (nanjiangshu) [reply proj/pred/templates/pred/submit_seq.html]: Looks great now!

**Review by nanjiangshu — APPROVED** [review]
LGTM!

**Pull request merged**